index.html

```diff
@@ -81,235 +81,215 @@
-      <p>The <code><dfn>crossorigin</dfn></code> <a href=
-      "http://www.w3.org/TR/html5/infrastructure.html#cors-settings-attributes">
-      CORS setting attribute</a> is an OPTIONAL attribute that indicates the
-      CORS policy of the specified resource. [[!HTML5]]</p>
-      <p>The <code><dfn>as</dfn></code> attribute on a <a>preload</a>
-      relationship specifies the <a href=
-      "https://fetch.spec.whatwg.org/#concept-request-context">request
-      context</a> used to initialize appropriate fetch settings - e.g. request
-      priority, HTTP headers, etc. [[!FETCH]]</p>
-      <pre class="example highlight html">
-&lt;link rel="preload" href="/assets/font.woff" as="font"&gt;
-&lt;link rel="preload" href="/assets/logo.webp" as="image"&gt;
-&lt;link rel="preload" href="//example.com/widget" as="iframe"&gt;
+    <pre class="example">
+  Link: &lt;https://example.com/other/styles.css&gt;; rel=preload; as=style
 </pre>
+    <p>As above examples illustrate, the resource can be specified via
+    declarative markup, Link HTTP header ([[!RFC5988]]), or scheduled via
+    JavaScript. See <a href='#use-cases'>use cases</a> section for more
+    hands-on examples of how and where <a>preload</a> can be used.</p>
+  </section>
+  <section>
+    <h2>Link type "<code>preload</code>"</h2>
+    <p>The <dfn data-lt="preload keyword">preload</dfn> keyword may be used
+    with [link] elements. This keyword creates an [external resource link]
+    (<dfn>preload link</dfn>) that is used to declare a resource and its fetch
+    properties. [[!HTML5]]</p>
+    <section>
+      <h2>Processing</h2>
+      <p>The <dfn>appropriate times</dfn> to <a>obtain the preload resource</a>
+      are:</p>
       <ul>
-        <li>The <code><a>as</a></code> attribute is a REQUIRED attribute for a
-        <a>preload</a> relationship. The specified <code><a>as</a></code> value
-        MUST be a valid <a href=
-        "https://fetch.spec.whatwg.org/#concept-request-context">request
-        context</a> as defined in [[!FETCH]]. If the <code><a>as</a></code>
-        attribute is omitted, or the specified value does not contain a valid
-        <a href="http://fetch.spec.whatwg.org/#concept-request-context">request
-        context</a>, then the user agent SHOULD output a developer-friendly
-        warning and ignore the <a>preload</a> relationship.
+        <li>When the user agent that supports [[!RFC5988]] processes `Link`
+        header that contains a <a>preload link</a>.
         </li>
-        <li>Request defaults set by the user agent via <code><a>as</a></code>
-        attribute MUST match the default settings set by the user agent when
-        processing a resource with the same context. This behavior is necessary
-        to guarantee correct prioritization and request matching - see
-          <a href="#matching-responses-with-requests"></a>.
+        <li>When the <a>preload link</a>'s [link] element is [inserted into a
+        document].
```

Comment:
Yes, that's covered by the other conditions below. Arguably we could be more explicit about this, but that's how HTML spec defines this behavior for other link's. I'm inclined to keep this as is.